pipelines/batch/src/main/java/org/openmrs/analytics/FetchResources.java

```diff
@@ -15,15 +15,18 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.Map;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 import ca.uhn.fhir.rest.api.SummaryEnum;
 import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.Maps;
 import org.apache.beam.sdk.transforms.DoFn.Element;
 import org.apache.beam.sdk.transforms.DoFn.ProcessElement;
```

Comment:
These packages do not seem to get used in here.

Reply:
Done.